testsuite/forge-cli/src/main.rs

```diff
@@ -429,6 +431,13 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
     let config =
         ForgeConfig::default().with_initial_validator_count(NonZeroUsize::new(30).unwrap());
     let single_test_suite = match test_name {
+        "epoch_changer_performance" => config
+            .with_network_tests(&[&PerformanceBenchmark])
+            .with_initial_validator_count(NonZeroUsize::new(5).unwrap())
+            .with_initial_fullnode_count(2)
+            .with_genesis_helm_config_fn(Arc::new(|helm_values| {
+                helm_values["chain"]["epoch_duration_secs"] = 60.into();
```

Comment:
Why hardcode epoch_duration_secs to be 60 secs?

Reply:
This is an epoch change stress test - so it aims to change the epoch every minute. I think that is fine to start with and we can tune it further going forward.